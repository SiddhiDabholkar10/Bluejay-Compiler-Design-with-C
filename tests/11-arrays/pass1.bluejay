arr[10] int := 10;
arr2[1] string := "string";


