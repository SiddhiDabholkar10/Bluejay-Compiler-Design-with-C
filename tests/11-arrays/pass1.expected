Array written "arr" type int
Array written "arr2" type string
